### ISSUE TITLE
Name conflict on hex.pm

### ISSUE THREAD

**Issue by jeanparpaillon** [aforward/dbus#2]
Hi,
I'm maintainning the [erlang-dbus](https://github.com/lizenn/erlang-dbus project) project, which is an erlang native implementation of dbus protocol. The project is already used in some other projects and the name come after the D-Bus protocol.

Since I've had a contrib for integrating erlang-dbus into hex.pm (from @ffringel), I figured out that the name of the erlang application conflicts with your app.

I'm really sorry for asking you this, but would it be possible to update the name of your app so that users looking for am erlang D-Bus implementation are not confused ?

Best regards,
Jean

**Comment by aforward** [comment]
Done 

https://github.com/aforward/sadbus

For HEX, let me me know which email address to add as an authority and you can push to overwrite the `dbus` hex.pm

**Comment by jeanparpaillon** [comment]
Thank you very much for this !

**Comment by ffringel** [comment]
Hi @aforward, thanks very much.
Here is my email address ffringel9@gmail.com. You can add that

**Comment by jeanparpaillon** [comment]
ffringel9@gmail.com has made the hex package and can push it. Thanks again

**Comment by aforward** [comment]
```
12:09 /src/sadbus (master)$ mix hex.owner add dbus ffringel9@gmail.com
A new Hex version is available (0.11.1), please update with `mix local.hex`
Adding owner ffringel9@gmail.com to dbus
```

Should be done.

**Comment by ffringel** [comment]
Awesome, and thanks again
